src/App.tsx

```diff
@@ -96,13 +95,13 @@ const App = () => {
 
   // hide splash screen
   useAsync(async () => {
-    const loggedOut = isRestored && !isBackedUp
-    const loggedInAndLoaded = isRestored && isBackedUp
+    const loggedOut = isRestored
+    const loggedInAndLoaded = isRestored
 
     if (loggedOut || loggedInAndLoaded) {
```

Comment:
Maybe we just change this to `if (isRestored)`

Reply:
ha, yeah.